### PR TITLE
EU-47

### PR DESCRIPTION
**Pull request by victor-nuzhniy**
Created endpoint for downloading student document

### CODE REVIEW

**Review by DmytroLyt — DISMISSED** [review]

File: apps/services/routers.py [hunk 1]
```diff
@@ -289,6 +289,35 @@ async def read_user_document(
...
+@services_router.get("/{university_id}/download-user-document/{user_document_id}",
+                     name="download_user_document",
+                     response_class=FileResponse,
+                     summary="Download user document",
+                     responses={200: {
+                         "description": "Successful download user document response",
+                         "content": {"text/html": {"example": "file"}}
+                     }},
+                     tags=["Student dashboard"])
+async def download_user_document(
```
Comment (anton-serozhechkin) [inline apps/services/routers.py:332]: Hi @victor-nuzhniy 
I can't open the downloaded file if I run the application inside the docker container. Can you please fix it?
Please see examples in ticket comments

File: apps/services/handlers.py [hunk 2]
```diff
@@ -270,5 +283,10 @@ async def __generate_user_document_name(cls, service_id: int, session: AsyncSess
         )
         return f"Заява на {service.service_name.lower()}"
 
+    @classmethod
```
Comment (anton-serozhechkin) [inline apps/services/handlers.py:293]: i added this method to separate some logic inside "download_user_document" method. Please don't forget to pull changes

File: apps/services/handlers.py [hunk 3]
```diff
@@ -270,5 +283,10 @@ async def __generate_user_document_name(cls, service_id: int, session: AsyncSess
...
+    async def __generate_user_document_name_for_download(cls, document_name: str, user_id: int, session: AsyncSession) -> str:
+        student = await student_service.read(session=session, data={"user_id": user_id})
+        return f"{document_name.replace(' ', '_')}_{student.first_name}_{student.last_name}.docx"
```
Comment (anton-serozhechkin) [inline apps/services/handlers.py:297]: f-string is better than str + str construction

**Review by anton-serozhechkin — CHANGES_REQUESTED** [review]

File: apps/services/routers.py (hunk 1 above)
Comment (victor-nuzhniy) [reply apps/services/routers.py:332]: Fixed it

File: apps/services/handlers.py (hunk 2 above)
Comment (victor-nuzhniy) [reply apps/services/handlers.py:293]: ok

File: apps/services/handlers.py (hunk 3 above)
Comment (victor-nuzhniy) [reply apps/services/handlers.py:297]: Ok

**Review by anton-serozhechkin — APPROVED** [review]

**Pull request merged**